### ISSUE TITLE
Fails with mypy>=0.780

### ISSUE THREAD

**Issue by kornicameister** [typeddjango/pytest-mypy-plugins#26]
I think I am raporting bug against correct repository.
Seems like something changed in mypy and with `mypy>=0.780` things are rather breaking up.
I am attaching a commit from my own repository [here](https://github.com/kornicameister/axion/commit/247cf04faac8cc157ad65923449654b597378ace).
You can checkout it or just go [here](https://app.circleci.com/pipelines/github/kornicameister/axion/1160/workflows/1d87a8de-170f-4361-90b4-24f0c50d5b96/jobs/3475/steps) to see how errors look like.

In short, it seems that something did change around how mypy deals with paths? Otherwise there wouldn't be any issue.
Something that was originally relative to the root of the project is not relative to something else making "expected" errors message impossible to deduce.

**Comment by kornicameister** [comment]
cc @sobolevn

**Comment by kornicameister** [comment]
Or, and sorry for being blind, perhaps that is related to something in `pydantic`.

**Comment by sobolevn** [comment]
`mypy>=0.782` works for me: https://github.com/dry-python/returns/pull/446

It hanleds correctly all the insane things we do.

**Comment by kornicameister** [comment]
Yeah, it was something inside of `pydantic`. Latest [release](https://github.com/kornicameister/axion/pull/356) fixed my own build.
Sorry for barking at a wrong tree :)
